wicket/src/wicketd.rs

```diff
@@ -113,11 +113,11 @@ async fn poll_inventory(
             match client.get_inventory().await {
                 Ok(val) => {
                     let new_inventory = val.into_inner();
-                    // TODO: We need the derive enhancements for progenitor to implement PartialEq/Eq
-                    //if new_inventory != inventory {
-                    //  inventory = new_inventory;
-                    //}
-                    let _ = tx.send(new_inventory).await;
+                    if new_inventory != inventory {
+                        inventory = new_inventory;
+                        let _ = tx.send(inventory.clone()).await;
+                        debug!(log, "No change to inventory");
```

Comment:
Should this log message be in an `else` block?

Reply:
oh, what the heck hahaha. Good catch!